info/v2/container.go

```diff
@@ -52,6 +52,14 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type ContainerInfo struct {
+	// Describes the container.
+	Spec ContainerSpec `json:"spec,omitempty"`
```

Comment:
No, I wasn't planning on it. This just adds a new method (`GetContainerInfoV2`) which will be called from kubelet for serving the new API at `/stats/summary`. I do not intend to make any changes to the other `/stats/...` endpoints.